### PR TITLE
feat(board): create `PBoard` component

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- <del>[ ] Tested with console</del> 
- Currently, `npm link` does not work between console and mirinae
- **Please check on console with released mirinae**

#### Create `PBoard` component.
- Created using `PBoardItem`.
- Update Docs.
- `pageLimit` props will be updated after console feature work.

---
- `pageLimit` props는 콘솔 작업을 진행하면서 사용성을 고려해 작업할 예정입니다.

screencapture-localhost-6006-2022-11-16-15_36_27

### CODE REVIEW

**Review by minseolee — APPROVED** [review]
LGTM! It looks very neat and efficient

File: src/data-display/board/PBoard.vue [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            default: 10,
+        },
+    },
+    setup(props, { emit }: SetupContext) {
+        const state = reactive({
+            boardList: computed<BoardSet[]>(() => props.boardSets),
+        });
+
+        const handleClickBoardItem = (item: BoardSet, index) => {
+            emit('click', item, index);
```
Comment (sulmoJ) [inline src/data-display/board/PBoard.vue]: I think it will overlap with the existing click event!

**Review by sulmoJ — APPROVED** [review]
I think it would be good if the list case that I mentioned verbally was added!
LGTM~

**Review by WANZARGEN — APPROVED** [review]

File: src/data-display/board/PBoard.vue (hunk 1 above)
Comment (piggggggggy) [reply src/data-display/board/PBoard.vue]: You are right. Edited to reflect your comments.

**Pull request merged**